.travis.yml

```diff
@@ -1,9 +1,6 @@
 language: ruby
 services:
   - docker
-rvm:
-  - 2.5
-  - 2.6
```

Comment:
Should we remove the CI builds for different versions of Ruby?  The builds for multiple versions let us see if anything broke between versions.

Multiple Build Example PR for reference: https://travis-ci.org/onc-healthit/inferno/builds/580890307

Reply:
This slows down the tests and I don't think we are currently gaining any value from it. I think the version being used in production is the only one that really matters.

Reply:
We've found some differences before with it, but if we're going to only use a single version of Ruby then I agree this is an unnecessary step.

Reply:
It might be worth adding in when we decide to upgrade to Ruby 2.6, but it's probably ok out for now.

Reply:
Travis respects your `.ruby-version` file, so we still won't need to specify a ruby version here.